tests/RLS/TableManagerTest.php

```diff
@@ -21,6 +21,7 @@
 use Stancl\Tenancy\Database\Exceptions\RecursiveRelationshipException;
 
 beforeEach(function () {
+    CreateUserWithRLSPolicies::$forceRls = true;
```

Comment:
Also reset this in afterEach

Reply:
Same thing in the other test files

Reply:
https://github.com/archtechx/tenancy/pull/1293/commits/a5c79d49c38d85ee26d3223fd89876309b8e27d3